charts/opentelemetry-operator/templates/prometheusrule.yaml

```diff
@@ -23,15 +23,15 @@ spec:
   - name: managerRules
     rules:
     - alert: ReconcileErrors
-      expr: rate(controller_runtime_reconcile_total{result="error"}[5m]) > 0
+      expr: rate(controller_runtime_reconcile_total{controller="opentelemetrycollector",result="error"}[5m]) > 0
```

Comment:
I'm not very familiar with Prometheus syntax, what is determining the value of `name`? Will it change based on the release name?

Reply:
@TylerHelmuth apparently the `name` label for the 2 metrics used in these alerts is hardcoded to `opentelemetrycollector`.

Reply:
@Allex1 hard coded in the Operator with those names?

Reply:
@TylerHelmuth yeah, in the metrics endpoint. Validated against my own metrics endpoint (which has a different helm release name), could not find it in the actual operator code.